README.md

```diff
@@ -102,18 +102,44 @@ val paymentFlowHandler = object : PaymentFlowHandler {
 }
 ```
 
-### Step 2: Create a configuration object for your Payment form
+### Step 2: Prepare your object responsible for the Frames configuration
 ```kotlin
+/*
+  PrefillData is purely optional. However, if you provide these details for your user you can
+        - make their checkout experience easier by prefilling fields they may need to do
+        - improve acceptance success for card tokenization
+ */
+val userPrefillData = PrefillData(
+    cardHolderName = "Test Name",        // represent the cardHolderName in PaymentDetail form
+    billingFormAddress = BillingFormAddress(
+        name = "UserName",               // represent the cardHolderName in Billing form
+        address = Address(
+            addressLine1 = "Hill road",
+            addressLine2 = "90 Tottenham Court Road",
+            city = "London",
+            state = "England",
+            zip = "W1T 4PY",
+            country = Country.from(iso3166Alpha2 = "GB")
+        ),
+        phone = Phone("7405987323", Country.UNITED_KINGDOM)
+    )
+)
+
+// Create a configuration object for your Payment form
 val paymentFormConfig = PaymentFormConfig(
     publicKey = PUBLIC_KEY,                     // set your public key
     context = context,                          // set context
     environment = Environment.SANDBOX,          // set the environment
     paymentFlowHandler = paymentFlowHandler,    // set the callback
     style = PaymentFormStyle(),                 // set the style
-    supportedCardSchemeList = emptyList()       // set supported card schemes, by default uses all schemes
+    supportedCardSchemeList = emptyList(),      // set supported card schemes, by default uses all schemes
+    prefillData = userPrefillData               // set prefill data for user to improve their checkout experience
 )
 ```
 
+#### Note for `cardHolderName` and `name` in `PrefillData`
+##### If `cardholderName` in the PaymentDetail form and `name` in the Billing form are pre-filled, then in the tokenization request, `cardHolderName` from the PaymentDetail form would be considered. Additionally, although the `cardHolderName` and `name` are pre-filled and fields are hidden in PaymentDetail form and Billing form, it will still be considering to pass in a tokenization request from the `PrefillData`.
```

Comment:
Let's change this to:

--

Please note that `cardholderName` in the payment form and `name` in the billing form represent the same info, so you should only include this either in the payment or billing form, not both. If you do include it in both places, we will prioritise passing `cardholderName` in the Tokenisation request. 

Additionally, if you hide these fields on the actual payment and billing forms, but include them in the `PrefillData` object, we will still pass them in the Tokenisation request. You may want to do this if you want to include this information in the Tokenisation request, but do not want the user to fill it out on the UI.

--

Also please remove the ##### here, and just make it normal text